libgit2-sys/lib.rs

```diff
@@ -2488,6 +2489,58 @@ extern {
                                    commit: *const git_oid, ancestor: *const git_oid)
                                    -> c_int;
 
+    // patch
+    pub fn git_patch_from_diff(out: *mut *mut git_patch,
+                               diff: *mut git_diff,
+                               idx: size_t) -> c_int;
+    pub fn git_patch_from_blobs(out: *mut *mut git_patch,
+                                old_blob: *const git_blob,
+                                old_as_path: *const c_char,
+                                new_blob: *const git_blob,
+                                new_as_path: *const c_char,
+                                opts: *const git_diff_options) -> c_int;
+    pub fn git_patch_from_blob_and_buffer(out: *mut *mut git_patch,
+                                          old_blob: *const git_blob,
+                                          old_as_path: *const c_char,
+                                          buffer: *const c_char,
+                                          buffer_len: size_t,
+                                          buffer_as_path: *const c_char,
+                                          opts: *const git_diff_options) -> c_int;
+    pub fn git_patch_from_buffers(out: *mut *mut git_patch,
+                                  old_buffer: *const c_char,
```

Comment:
The systest checkers are failing and it looks like this argument is `c_void` I think?